.github/workflows/build_release_candidates.yml

```diff
@@ -10,6 +10,8 @@ on:
       - master
 
 jobs:
+  windows:
+    uses: ./.github/workflows/build_windows_release_candidate.yml
```

Comment:
oh, neat, didn't know you could do this!

Reply:
Right, we can do similar things for other pipelines as well.

Reply:
with this we could put our workflow files in our workspace modules, if we wanted to

Reply:
btw, this uploads the artifact to the triggering workflow, right?

Reply:
Yes, you can see the artifacts here: https://github.com/jyyi1/outline-client/actions/runs/1815112284, we have both Linux and Windows artifacts.

Reply:
sweet!!